### ISSUE TITLE
Node installed  wu-wei dependency for the benchmark interferes with wu-wei

### ISSUE THREAD

**Issue by dherre3** [Sable/bfs-benchmark#3]
**Description**:
The test folder in the wu-wei dependency for the benchmark interferes with the wu-wei folder where the actual benchmark is hosted. 
The following benchmarks are present in that test folder:
- backprop-benchmark
- fib
- gcc

The benchmark runs fine under normal conditions, the problem becomes when trying to do an incorrect configuration with the benchmark.

**Steps to Reproduce**:
1. `wu run bfs c -v`
These makes the original wu-wei look into the dependency wu-wei folder for the c implementation. It then finds the fib implementation and since that implementation is not valid it throws,
```
clang: error: no such file or directory: '/Users/davidherrera/Documents/Research/repos-wu-wei/wu-sable/wu-wei-benchmarking-toolkit/benchmarks/bfs/node_modules/wu-wei-benchmarking-toolkit/tests/public/fib/implementations/c/imagenet.c'
clang: error: no such file or directory: '/Users/davidherrera/Documents/Research/repos-wu-wei/wu-sable/wu-wei-benchmarking-toolkit/benchmarks/bfs/node_modules/wu-wei-benchmarking-toolkit/tests/public/fib/implementations/c/backprop.c'
clang: error: no such file or directory: '/Users/davidherrera/Documents/Research/repos-wu-wei/wu-sable/wu-wei-benchmarking-toolkit/benchmarks/bfs/node_modules/wu-wei-benchmarking-toolkit/tests/public/fib/implementations/c/common/common.c'
clang: error: no such file or directory: '/Users/davidherrera/Documents/Research/repos-wu-wei/wu-sable/wu-wei-benchmarking-toolkit/benchmarks/bfs/node_modules/wu-wei-benchmarking-toolkit/tests/public/fib/implementations/c/common/common_rand.c
```

**Expected Behaviour**:
Display the current setups available.

**Comment by elavoie** [comment]
Thanks for reporting, I'll fix it.

**Comment by elavoie** [comment]
Fixed by Sable/wu-wei-benchmarking-toolkit@73db664e301ece4069af3be41736703915b7a2a4
